src/string.js

```diff
@@ -137,8 +175,8 @@ inherits(StringSchema, MixedSchema, {
   },
 
   uppercase(message = locale.uppercase) {
-    return this.transform(
-      value => (!isAbsent(value) ? value.toUpperCase() : value),
+    return this.transform(value =>
+      !isAbsent(value) ? value.toUpperCase() : value,
```

Comment:
The husky pre-commit hook caught some style errors, so I thought I'd land them in this commit too. I can also disable the precommit hook and retry if this is too noisy